### PR TITLE
docs(rating): add API docs

### PR DESCRIPTION
**Pull request by pkozlowski-opensource**

### CODE REVIEW

File: src/rating/rating.ts [hunk 1]
```diff
@@ -17,12 +20,37 @@ export class NgbRating implements OnInit {
...
   @Input() max: number = 10;
+
+  /**
+   * Current rating.
+   */
   @Input() rate: number;
-  @Input() readonly: boolean;
 
+  /**
+   * A flag indicating if rating could be read only (true) or updated (false).
```
Comment (Foxandxss) [inline src/rating/rating.ts]: I find this a bit verbose. I don't think we need to explain what true or false means in this case. I mean, bit obvious from the name and the type of the input (boolean).

File: src/rating/rating.ts [hunk 2]
```diff
@@ -17,12 +20,37 @@ export class NgbRating implements OnInit {
...
+   */
+  @Input() readonly = false;
+
+  /**
+   * An event fired when a user is hovering over a given rating.
+   * Event's payload equals to the rating being hoovered over.
```
Comment (Foxandxss) [inline src/rating/rating.ts]: hoovered, that is a typo.

File: src/rating/rating.ts [hunk 3]
```diff
@@ -17,12 +20,37 @@ export class NgbRating implements OnInit {
...
+   */
   @Output() hover = new EventEmitter<number>();
+
+  /**
+   * An event fired when a user stops hovering over a given rating.
+   * Event's payload equals to the rating of the last item being hoovered over.
```
Comment (Foxandxss) [inline src/rating/rating.ts]: ditto

File: src/rating/rating.ts (hunk 1 above)
Comment (pkozlowski-opensource) [inline src/rating/rating.ts]: Could propose an alternative wording?

Comment (Foxandxss) [inline src/rating/rating.ts]: What about...

"A flag indicating whether the rating is read only"

Probably @icfantv knows more english than we do.

Comment (icfantv) [inline src/rating/rating.ts]: i'm ok w/ @Foxandxss' proposal.

Comment (wesleycho) [inline src/rating/rating.ts]: Or how about `A flag indicating if rating can be updated`. I think this is actually a little better at communicating the intent

**Pull request closed without merging**